app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -13,9 +13,39 @@ def orders_volunteers
     @orders = Volunteer.find(params[:id]).orders
   end
 
+  def rating
+    @order_request = OrderRequest.find_by("order_id = ? AND order_request_status = ?", params[:order_id], OrderRequest.order_request_statuses[:accepted])
+    if @order_request != nil then 
+      @rating = VolunteerRating.new
+      @rating.order_id = params[:order_id]
+      @rating.qualifier_id = @order_request.volunteer.id
+      @rating.qualified_id = @order_request.order.helpee.id
+      @rating.score = params[:score]
+      @rating.comment = params[:comment]
+      @rating.save!
+
+      @order_request.order.updated_at = Time.now
+      @order_request.order.save
+      head :ok
+    else
+      head :bad_request
+    end
+  end
+
+  def rating_pending
+    @volunteer_id = params[:volunteer_id]
```

Comment:
```suggestion
    volunteer_id = params[:volunteer_id]
```
Mismo que el otro controlador.